hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/security/TestOzoneBlockTokenSecretManager.java

```diff
@@ -328,7 +328,7 @@ public void testNetYetValidCertificate() throws Exception {
   private X509Certificate generateExpiredCert(String dn,
       KeyPair pair, String algorithm) throws CertificateException,
       IllegalStateException, IOException, OperatorCreationException {
-    Date from = new Date();
+    Date from = new Date(Instant.now().toEpochMilli() - 100L);
     Date to = new Date(from.getTime() + 100L);
     return generateTestCert(dn, pair, algorithm, from, to);
```

Comment:
Can we just do -
`` generateTestCert(dn, pair, algorithm, from, from);``
Means eliminate the `to` and pass `from` only instead, rather than this?

Reply:
Thanks for the suggestion, have set endDate same as startDate.